raiden/tests/utils/tester_client.py

```diff
@@ -602,13 +602,15 @@ def deposit(self, amount):
         self.proxy.deposit(amount)
         self.tester_state.mine(number_of_blocks=1)
 
+    # opened() and closed() method return different values for geth and tester chain
```

Comment:
Tester client is only used for tester chain. Which failure are you trying to solve? Can you show us a reproducible scenario?